### PR TITLE
📖 Some overdue documentation updates

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR does some catch-up on the existing document set.

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/v0.20/release.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Making KubeStellar Releases
 
-This document defines how releases of the KubeStellar repository are made.
+This document defines how releases of the KubeStellar repository are made. This document is a work-in-progress. In particular, the dependency cycle between the `kubestellar` and `ocm-tansport-plugin` repos is not well documented and we have not agreed on how to test and release in the presence of this cycle.
```
Comment (ezrasilvera) [inline docs/content/v0.20/release.md:3]: nit: Maybe replace the "we have not agreed"  ? 
Maybe " ....repos is not well documented and  it's still not clear how to test and release in the presence of this cycle. "

File: docs/content/v0.20/README.md [hunk 2]
```diff
@@ -48,8 +47,18 @@ See [Packaging and Delivery](packaging.md)
 
 ## Usage examples, and testing
 
-See [Examples](examples.md) and [integration testing](contributor.md#integration-testing) for _this_ release, after reviewing [above](#latest-stable-release).
+[Examples](examples.md) shows how to use a release of KubeStellar.
```
Comment (ezrasilvera) [inline docs/content/v0.20/README.md:50]: Maybe:
shows how to install the KubeStellar environment and describe the step by step instructions for few usage examples. 

Why is it important to explicitly mention "the release"  ?  All the documentation (except the developer section that show how to build and test the code) is expected to use releases

File: docs/content/v0.20/contributor.md [hunk 3]
```diff
@@ -59,6 +59,12 @@ X ArgoCD CLI
   how to install: https://argo-cd.readthedocs.io/en/stable/cli_installation/
 ```
 
+## Unit testing
+
```
Comment (ezrasilvera) [inline docs/content/v0.20/contributor.md:63]: Do we want to add a line of description (e.g., In order to run the unit tests ......)

File: docs/content/v0.20/packaging.md [hunk 4]
```diff
@@ -144,12 +136,17 @@ The dashed dependencies are at run time, not build time.
 
 "KCM" is the KubeStellar controller-manager.
 
-"ks_scripts" are the user-facing instructions that use published
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:147]: Not part of this PR, but I don't think that anyone is using this acronym (KCM)   for the few places we refer to it we can go with "KS controller maanger"

**Comment by MikeSpreitzer** [comment]
The force-push to a80dca6fc makes some changes in response to the review above.

File: docs/content/v0.20/release.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/release.md]: done

File: docs/content/v0.20/README.md (hunk 2 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/README.md]: I also added a little elaboration here, like the suggested additional information.

File: docs/content/v0.20/contributor.md (hunk 3 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/contributor.md:63]: done

File: docs/content/v0.20/packaging.md (hunk 4 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:147]: Don't forget to think about the verbosity of the labels in the diagrams.
